website/docs/error-reference.md

```diff
@@ -4058,6 +4058,13 @@ To fix this error, ensure that the left and right operands' types match before
 doing the comparison. For example, try converting `String`s to `Symbol`s with
 `to_sym` (or vice versa with `to_s`).
 
+## 7047
+
+Usages of values with type `T.untyped` are highlighted by this error. It is
+intended to only show up as a warning in editors through LSP for files with
+typed sigil set between `true` and `strong` (exclusive). For `typed: strong`
```

Comment:
Suggestion: could write this as "typed sigil set to `true` or `strict`" (I'm okay with the current phrasing if you prefer that).